sdcm/cluster_aws.py

```diff
@@ -675,6 +675,12 @@ def restart(self):
             try:
                 self.stop_scylla_server(verify_down=False)
 
+                # Moving var-lib-scylla.mount away, since scylla_create_devices fails if it already exists
```

Comment:
var-lib-scylla.mount file is created by scylla_setup.
what's the error if var-lib-scylla.mount exists?

If the var-lib-scylla.mount is moved, will the RAID be mounted correctly after reboot?

Reply:
We don’t know, isn’t it created by create_devices?

We saw that create devices is failing when this file exist

Reply:
seem like @syuu1228 did this change few months ago:
https://github.com/scylladb/scylla/commit/c0b2933106c2274720c82e9637bb0e15f54fc324

I think we should remove it file inside create_devices script, if that's the case

Reply:
> We don’t know, isn’t it created by create_devices?
> 
> We saw that create devices is failing when this file exist

In restart() we call scylla_create_devices script to setup RAID, the existing var-lib-scylla.mount won't effect it.
it only effects scylla_setup script, but we didn't call it in restart().

Reply:
> seem like @syuu1228 did this change few months ago:
> [scylladb/scylla@c0b2933](https://github.com/scylladb/scylla/commit/c0b2933106c2274720c82e9637bb0e15f54fc324)
> 

The commit changed scylla_setup, but we won't call it in restart().

> I think we should remove it file inside create_devices script, if that's the case

@ShlomiBalalis Need to check the source of the error someone touched, is it from scylla_setup, why it's called.

Currently we only directly call scylla_setup script for GCE backend in init stage.
A flag file `/etc/scylla/machine_image_configured` will be created after `scylla_setup -> scylla_image_setup` succeeds, then scylla-image-setup won't be called again.